brick/hooks/pubspec.yaml

```diff
@@ -0,0 +1,10 @@
+name: very_good_wear_app_hooks
+
+environment:
+  sdk: ">=2.17.0 <3.0.0"
```

Comment:
on the previous PR the minimum is 2.18, is ok to have different versions?

Reply:
I think it should be bumped to 2.18.0